raiden/transfer/mediated_transfer/transition.py

```diff
@@ -12,10 +12,10 @@ def update_route(next_state, route_state_change):
 
     # TODO: what if the route that changed is the current route?
 
-    if new_route.state == 'unavailable':
+    if new_route.state != 'opened':
         available_routes.pop(available_idx)
 
-    elif new_route.state == 'available':
+    elif new_route.state == 'opened':
```

Comment:
we're using these "magic" state strings in a ton of places. usually I am not a fan of doing it, but maybe here it could make sense to have some constants for those?